src/main/java/edu/harvard/iq/dataverse/api/BuiltinUsers.java

```diff
@@ -105,7 +105,28 @@ public Response create(BuiltinUser user, @PathParam("password") String password,
         return internalSave(user, password, key);
     }
     
+    //TODO: detailed description 
+    /**
+     * Created this for #6915...
+     *
+     * @param user
+     * @param password
+     * @param key
+     * @param sendEmailNotification
+     * @return
+     */
+    @POST
+    @Path("{password}/{key}/{sendEmailNotification}")
```

Comment:
This new endpoint should be documented. Here's a link to the current docs in this area: http://guides.dataverse.org/en/4.20/api/native-api.html#create-a-builtin-user

The file to edit is doc/sphinx-guides/source/api/native-api.rst